modules/vector-search/pages/fine-tune-vector-search.adoc

```diff
@@ -0,0 +1,132 @@
+= Fine-tuning Vector Searches
+:page-topic-type: concept
+:page-ui-name: {ui-name}
+:page-product-name: {product-name}
+:stem:
+:description: pass:q[Additional parameters can be added to a vector search REST call that will allow you to tune the search for recall and/or accuracy.]
+
+[abstract]
+{description}
+
+[#overview]
+== Overview
+
+Couchbase automatically dynamically tunes vector indexes to achieve a balance between recall (the quality of the search results),
+latency (search response time), and memory efficiency.
+This tuning occurs during indexing and querying, eliminating the need for users to adjust optimization parameters manually.
+
+Couchbase dynamically adjusts two critical vector parameters:
+
+`nlist` (or `Centroid` count)::
+The number of clusters used for indexing.
+The centroids are used to quickly find the surrounding closest matches in the index. Increasing the number of centroids will increase accuracy but will decrease the speed of the search.
++
+The `nlist` is determined dynamically based on the size of the dataset (number of vectors) in a partition:
++
+[%header, cols="3*a"]
+|===
+| Number of vectors in partition (`nvec`)
+| `nlist` calculation
+| Notes
+
+| stem:["nvec " ge " 200,000"]
+| stem:[4 xx sqrt("nvec")]
+| This formula is designed to handle larger datasets
+ where increasing the number of datasets does not yield significant improvements in recall.
+
+| stem:["1000" lt "nvec" le "200,000" ]
+| stem:["nvec" / 100]
+| This formula targets approximately 100 vectors per cluster,
+which balances between too few and too many clusters, ensuring efficient indexing.
+
+| stem:["nvec" lt 1000]
+| N/A
+| For number of vectors less than 1000, Couchbase will carry out a straight forward one-to-one mapping between IDs and vectors with an exact vector comparison.
+Vectors are directly stored without the need for additional processing for the `nlist` calculation.
+
+|===
+
+`nprobes` (or `probes`)::
+The number of `centroids` that a query will check for similar vectors.
+The `nprobe` value is only set when Couchbase is using an Inverted File Index. Couchbase will select the best index type and comparison method depending on the size of the dataset and the `vector_index_optimized_for` setting.
```

Comment:
```suggestion
The `nprobe` value is only set when the Search Service is using an Inverted File Index. The Search Service will select the best index type and comparison method depending on the size of the dataset and your `vector_index_optimized_for` setting.
```
Please add a link to where we explain the `vector_index_optimized_for` setting. I know it's discussed in creation topics.